### DISCUSSION TITLE
📔【操作系统】锁的实现 🆕

### DISCUSSION THREAD

**Discussion by imageslr** [imageslr/blog#89] [Announcements]
https://imageslr.github.io/2020/locks.html 

 互斥锁的实现 1. 禁止中断 进入临界区前禁止中断，离开之前恢复中断。这样任何中断都不会发生，包括时钟中断，也就是说 CPU 不会被切换到其他线程。

**Comment by Vector6662** [comment]
博主第二个应该是TSL吧？笔误

**Comment by imageslr** [comment]
@Vector6662  谢谢，已改正。
